src/site/asn1.rst

```diff
@@ -69,6 +69,28 @@ Part 2 --- Country Catalogue
 ============================
 
 
+#. Create a method called ``setPopulation`` that takes a name of a country and a new population value for that country as parameters
+
+#. Have a method called ``numberCountriesOnContinent`` that takes a continent name as a parameter and returns the number of ``Country`` objects on that continent
+
+#. Write a method called ``filterCountriesByContinent`` that takes a continent name as a parameter and returns a new ``CountryCatalogue`` instance containing only the countries on the specified continent
```

Comment:
I was gonna' have a note about having an instance of `CountryCatalogue` making a new instance of a `CountryCatalogue`, but I opted to not bother.